packages/modeling/src/operations/extrusions/extrudeLinear.js

```diff
@@ -1,8 +1,9 @@
 const flatten = require('../../utils/flatten')
 
-const geom2 = require('../../geometries/geom2')
+const { geom2, path2 } = require('../../geometries')
```

Comment:
Not wrong but we try not to use the indexes when requiring specific objects. In the past, there have been circular references which node couldn't handle. Can you split this into two (2) requires statements?